doc/user/content/connect-sources/postgres-amazon-rds.md

```diff
@@ -0,0 +1,395 @@
+
+Once logical replication is enabled, the next step is to create a publication with the tables that you want to replicate to Materialize. You'll also need create a user for Materialize and give the user permissions to manage replication.
+
+1. For each table that you want to replicate to Materialize, set the [replica identity](https://www.postgresql.org/docs/current/sql-altertable.html#SQL-ALTERTABLE-REPLICA-IDENTITY) to `FULL`:
+
+    ```sql
+    ALTER TABLE table1 REPLICA IDENTITY FULL;
+    ```
+
+    This setting makes sure sufficient information is written to the replication stream for  `UPDATE` and `DELETE` operations.
+
+    <!-- THE LINKED PAGE DOESN'T TALK ABOUT INCREASED CPU. WE SHOULD FIND ANOTHER PAGE TO LINK TO. COMMENTING THIS OUT IN THE MEANTIME. As a heads-up, you should expect a performance hit in the database from increased CPU usage. For more information, see the [PostgreSQL documentation](https://www.postgresql.org/docs/current/logical-replication-publication.html). -->
+
+1. Create a [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) with the tables you want to replicate:
+
+    _For specific tables:_
+
+    ```sql
+    CREATE PUBLICATION mz_source FOR TABLE table1, table2;
+    ```
+
+    _For all tables in the database:_
+
+    ```sql
+    CREATE PUBLICATION mz_source FOR ALL TABLES;
+    ```
+
+    The `mz_source` publication will contain the set of change events generated from the specified tables, and will later be used to ingest the replication stream.
+
+    We strongly recommend that you **limit** publications only to the tables you need.
+
+1. Create a user for Materialize:
+
+    ``` sql
+    CREATE USER materialize PASSWORD '<password>';
+    ```
+
+1. Grant the user permission to manage replication:
+
+    ``` sql
+    GRANT rds_replication TO materialize;
+    ```
+
+1. Grant the user the required permissions on the tables you want to replicate:
+
+    ```sql
+    GRANT CONNECT ON DATABASE <dbname> TO materialize;
+    ```
+
+    ```sql
+    GRANT USAGE ON SCHEMA <schema> TO materialize;
+    ```
+
+    ```sql
+    GRANT SELECT ON ALL TABLES IN SCHEMA <schema> TO materialize;
+    ```
+
+    **Note:** `SELECT` privileges on the tables you want to replicate are needed for the initial snapshot.
+
+## Step 3. Configure your network
+
+When your PostgreSQL database is running in a private network, there are various ways to allow Materialize to connect:
+
+- **Allow Materialize IPs:** This is the simplest approach. You configure your database's security group to allow public connections from a set of static Materialize IP addresses and rely on PostgreSQL authentication and authorization to keep your data safe.
+
+- **Use AWS PrivateLink** or **Use an SSH tunnel:** If you need an additional layer of security at the network level, you can use either [AWS PrivateLink](https://aws.amazon.com/privatelink/) or an SSH tunnel to give Materialize secure access to your database.
```

Comment:
The phrasing here is weird. It's not about adding an additional layer of security or give Materialize secure access, but more about being able to connect to your database if it's in a private network.

Reply:
Done. I've removed mention of levels of security for now. @josharenberg, since you gave me some help on that language, let me know if you think it's particularly important.

Reply:
From a cloud security perspective, the `Allow Materialize IPs` is the least secure of these options, which may be what the phrasing is hinting at. Some infra teams will not be ok with allowlisting IPs. However, this option would not work unless the database is [already in a public subnet and publicly accessible](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_VPC.WorkingWithRDSInstanceinaVPC.html#USER_VPC.IP_addressing), so it might make more sense just to include those prerequisites in the option rather than mentioning security. Presumably a cloud team that is running their database as publicly accessible has already considered the security tradeoff!
Perhaps:
>- **Allow Materialize IPs:** This is the simplest approach, but requires that your database is configured as publicly accessible.

Reply:
Rephrased.